### PR TITLE
style: Final colour simplification

### PR DESCRIPTION
**Pull request by mgrhm**
This pull further simplifies the colour palette from 25 colours to 19.  It simplifies the palette and code.  Once this is merged, I'll be able to start work on the dark mode (#1192).

There are quite a lot of small, separate commits in this.  A summary of the main changes:

- Six near-neighbour or one-use colours removed
- All hex code colours are now six character
- All word-named colours are now hex codes
- All RGBA colours are now hex codes

### CODE REVIEW

**Review by asbiin — APPROVED** [review]
I think that's really nice like that!
Don't you want to create variables for every colors in 1 file, and use these variables everywhere?

**Comment by mgrhm** [comment]
I'd planned to do exactly that in the 'final' colour commit where I implement a dark mode.

**Comment by mgrhm** [comment]
Hi @asbiin.  I've removed the compiled CSS and JSON assets from this pull request.

Do you need me to do anything else before you can merge?

File: resources/sass/people.scss [hunk 1]
```diff
@@ -249,17 +249,17 @@
...
       margin-bottom: 25px;
       padding: 10px;
       position: relative;
       font-size: 0.875rem;
 
       &.edit {
-        background-color: #fffceb;
-        border-color: gold;
+        background-color: #fffacd;
+        border-color: #daa520;
```
Comment (asbiin) [inline resources/sass/people.scss:262]: These colors are near, but the final aspect is quite different. Could we find closer colors as the originals?

**Review by asbiin — CHANGES_REQUESTED** [review]
Great job @mgrhm ! Hope to see the next one;)
See my comment here:

File: resources/sass/people.scss (hunk 1 above)
Comment (mgrhm) [reply resources/sass/people.scss]: I’ve restored these two to their originals.

**Review by asbiin — APPROVED** [review]
👍🏽

**Comment by asbiin** [comment]
@mgrhm Thank you so much, your changes are much appreciate

**Pull request merged**